source/designers-guide/javascript-coding-style/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: JavaScript Coding Style
+github_link: designers-guide/javascript-coding-style/index.md
+indexed: true
+---
+
+This document attempts to explain the basic styles and patterns used in the Shopware JavaScript codebase. New code should try to conform to these standards so that it is as easy to maintain as existing code. Of course every rule has an exception, but it's important to know the rules nonetheless!
```

Comment:
This document explains the basic styles and patterns used in the Shopware JavaScript codebase. New code should try to conform to these standards so that it is as easy to maintain as existing code. Of course every rule has an exception, but it's important to know the rules nonetheless.